gramex/gramex.yaml

```diff
@@ -216,6 +216,18 @@ log:
 eventlog:
     path: $GRAMEXDATA/logs/events.db          # Location of application event logs.
 
+gramexlog:
+    default:
+        host: $ES_HOST
+        user: $ES_USER
+        pass: $ES_PASS
+        index: gramexlog
+    app:
```

Comment:
Remove this from `gramex/gramex.yaml` but add it to `tests/gramex.yaml` with a different hostname (e.g. `localhost` vs `127.0.0.1`)